public/assets/css/blogpost.css

```diff
@@ -0,0 +1,206 @@
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 95%;
+  min-height: 100vh;
+}
+header {
+  width: 100%;
+}
+
+.card-wrap {
+  width: 100%;
+}
+
+.blog-card {
+  padding: 60px;
+  max-width: 1100px;
+  margin: auto;
+  margin-top: 50px;
+  color: var(--text-secondary);
+}
+
+.blog-card h1,
+.blog-card h2,
+.blog-card h3,
+.blog-card h4,
+.blog-card h5,
+.blog-card h6,
+.blog-card strong,
+.blog-card a,
+.blog-card a * {
+  color: white;
+}
+
+.blog-card .title {
+  margin: 0;
+  margin-bottom: 8px;
+}
+
+.blog-card .pub-info {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: left;
+  margin-top: auto;
+  margin-bottom: 30px;
+}
+.blog-card .pub-info .date {
+  font-weight: bold;
+  color: var(--text);
+}
+.blog-card .pub-info > * {
+  margin-right: 0.5em;
+  margin-top: 0.2em;
+}
+.blog-card .profile {
+  display: inline-grid;
+  grid-template-columns: 30px auto;
+  grid-gap: 10px;
+  font-weight: bold;
+  color: var(--text);
+  align-items: center;
+  height: 32px;
+}
+.blog-card .profile img {
+  margin: 0;
+  border-radius: 4px;
+  border: 1px solid rgba(255, 255, 255, 0.1);
+  max-width: 100%;
+}
+
+.blog-card p,
+.post-info {
+  color: var(--text-secondary);
+}
+
+.blog-card img {
+  max-width: 100%;
+  max-height: 800px;
+  margin: 10px auto;
+  display: block;
+  border-radius: 4px;
+  border: 1px solid rgba(255, 255, 255, 0.1);
+}
+.blog-card img.emoji {
+  display: inline;
+  margin: 0;
+  border: none;
+}
+
+.blog-card video {
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid rgba(255, 255, 255, 0.1);
+}
+
+.blog-card iframe {
+  width: 100%;
+  aspect-ratio: 16/9;
+  border-radius: 4px;
+  border: 1px solid rgba(255, 255, 255, 0.1);
```

Comment:
I think we should make the `rgba(255,255,255,0.1)`s everywhere into custom properties. `--border` will do.